files/slave.dockerfile

```diff
@@ -2,21 +2,20 @@ FROM docker.io/stackbrew/centos:latest
 MAINTAINER Chris Evich <cevich@redhat.com>
 
 RUN yum update -y && \
-    yum install -y findutils epel-release https://rdoproject.org/repos/rdo-release.rpm && \
+    yum install -y epel-release https://rdoproject.org/repos/rdo-release.rpm && \
```

Comment:
Nope, I just moved it into ``files/slave.rpms``.  This won't affect the test VMs, remember slave == kommandir-container (and needs to be renamed).  i.e. when everything is up and going, the topology looks like this:

```
executing node (e.g. Jenkins) -> kommandir vm -> kommandir container -> peons (test VMs)
```